src/main/java/org/reaktivity/nukleus/oauth/internal/OAuthConfiguration.java

```diff
@@ -21,18 +21,24 @@
 {
     public static final String KEYS_NAME = "nukleus.oauth.keys";
     public static final String AUTO_DISCOVER_REALMS_NAME = "nukleus.oauth.auto.discover.realms";
+    public static final String CHALLENGE_DELTA_CLAIM_NAMESPACE_NAME = "nukleus.oauth.namespace";
+    public static final String CHALLENGE_DELTA_CLAIM_NAME_NAME = "nukleus.oauth.name";
```

Comment:
Each constant name should derive algorithmically from its property name.
Property names: `nukleus.oauth.claim.namespace`, `nukleus.oauth.claim.name.challenge.response.timeout`.